skore/src/skore/item/cross_validation_item.py

```diff
@@ -46,6 +46,14 @@ def plot_cross_validation(cv_results: dict) -> plotly.graph_objects.Figure:
 
     df = pandas.DataFrame(_cv_results)
 
+    # Move time columns to last and "test_score" to first
```

Comment:
doing these manipulations on the dataset seem a bit weird to me, isn't there a simpler way by using the [category_orders](https://plotly.com/python/legend/) function of plotly?

Reply:
I didn't know about `category_orders`, thanks! Although, I can't find a way to use it in this case because it applies to categorical columns.
If you can see an easy way to do it, do let me know.

On the other hand, the dataframe only exists in the context of the function, so there should be no safety concerns.

Reply:
ok, then it's fine for me :) !